vm/ubpf_vm.c

```diff
@@ -290,6 +290,7 @@ ubpf_exec(const struct ubpf_vm* vm, void* mem, size_t mem_len, uint64_t* bpf_ret
     uint64_t* reg;
     uint64_t _reg[16];
     uint64_t ras_index = 0;
+    int return_value;
```

Comment:
I know that this is annoying, but I think it would be reassuring if we had an initial value in here in case we add uses later where we don't especially set the value before `goto` cleanup.

Reply:
Agreed.